docs/contribute/01_environment.qmd

```diff
@@ -67,37 +74,48 @@ for manager, params in managers.items():
     print("    gh repo fork --clone --remote ibis-project/ibis")
     print("    ```")
     print()
-    print(f"1. Create a Conda environment using `environment.yml`")
+    print(f"1. Create a {name} environment using `environment.yml`")
     print()
     print("    ```sh")
     print("    cd ibis")
-    print(f"    {manager} env create --file conda/environment.yml")
+    if manager == "conda" or manager == "mamba":
+        print(f"    {manager} env create --file conda/environment.yml")
+    elif manager == "pixi":
+        print(f"    {manager} init --import conda/environment.yml")
     print("    ```")
     print()
-    if manager == "conda" or manager == "mamba":
-        print()
-        print(
-            "    If you're using arm64 architecture (Mac M1/M2), use "
-            "`conda/environment-arm64.yml` for setting up a dev environment for all the "
-            "backends that are possible to install excluding Flink; use "
-            "`conda/environment-arm64-flink.yml` for setting up a dev environment for all "
-            "the backends that are possible to install including Flink. The reason to have "
-            "two separate environments is because `apache-flink` forces `pyarrow` to "
-            "downgrade to 11.0, which causes conflicts in other backends."
-        )
+    print(
+        "    If you're using arm64 architecture (Mac M1/M2), use "
+        "`conda/environment-arm64.yml` for setting up a dev environment for all the "
+        "backends that are possible to install excluding Flink; use "
+        "`conda/environment-arm64-flink.yml` for setting up a dev environment for all "
+        "the backends that are possible to install including Flink. The reason to have "
+        "two separate environments is because `apache-flink` forces `pyarrow` to "
+        "downgrade to 11.0, which causes conflicts in other backends."
+    )
+    print()
+    if manager == "pixi":
+        print("    ::: {.callout-note}")
+        print("    This command will initialize a `pixi.toml` file and also modify ")
```

Comment:
Alternatively, we could provide Pixi configuration in `pyproject.toml` (or `pixi.toml`), but this would be more involved to keep in sync; this approach worked well enough for me so far, with minimal effort.